docs/available_assets.md

```diff
@@ -0,0 +1,233 @@
+<i class="fas fa-exclamation-triangle"></i> required asset: *none*  
+<i class="fas fa-exclamation-circle"></i> required software: *none*
+
+The `ensembl_gtf` asset is used to build other derived assets including a comprehensive TSS annotation and gene body annotation.
+
+Example Ensembl files:
+
+- [hg38 ensembl annotations](ftp://ftp.ensembl.org/pub/current_gtf/homo_sapiens/Homo_sapiens.GRCh38.98.gtf.gz)
+- [hg19 ensembl annotations](ftp://ftp.ensembl.org/pub/grch37/current/gtf/homo_sapiens/Homo_sapiens.GRCh37.87.gtf.gz)
+- [mm10 ensembl annotations](ftp://ftp.ensembl.org/pub/current_gtf/mus_musculus/Mus_musculus.GRCm38.98.gtf.gz)
+- [rn6 ensembl annotations](ftp://ftp.ensembl.org/pub/current_gtf/rattus_norvegicus/Rattus_norvegicus.Rnor_6.0.98.gtf.gz)
+
+```
+wget ftp://ftp.ensembl.org/pub/release-97/gtf/homo_sapiens/Homo_sapiens.GRCh38.97.gtf.gz
+refgenie build hg38/ensembl-gtf --ensembl_gtf Homo_sapiens.GRCh38.97.gtf.gz
+```
+
+### ensembl_rb
+
+<i class="fas fa-exclamation"></i> required input: `--gff` [regulatory_features.gff.gz](http://useast.ensembl.org/info/genome/funcgen/regulatory_build.html)  
+<i class="fas fa-exclamation-triangle"></i> required asset: *none*  
+<i class="fas fa-exclamation-circle"></i> required software: *none*
+
+The `ensembl_rb` asset is used to produce derived assets including feature annotations.
+
+Example Ensembl files:
+
+- [hg38 regulatory build](ftp://ftp.ensembl.org/pub/current_regulation/homo_sapiens/homo_sapiens.GRCh38.Regulatory_Build.regulatory_features.20190329.gff.gz)
+- [hg19 regulatory build](ftp://ftp.ensembl.org/pub/grch37/current/regulation/homo_sapiens/homo_sapiens.GRCh37.Regulatory_Build.regulatory_features.20180925.gff.gz)
+- [mm10 regulatory build](ftp://ftp.ensembl.org/pub/current_regulation/mus_musculus/mus_musculus.GRCm38.Regulatory_Build.regulatory_features.20180516.gff.gz)
+
+```
+wget ftp://ftp.ensembl.org/pub/current_regulation/homo_sapiens/homo_sapiens.GRCh38.Regulatory_Build.regulatory_features.20190329.gff.gz
+refgenie build hg38/ensembl_rb --gff homo_sapiens.GRCh38.Regulatory_Build.regulatory_features.20190329.gff.gz
+```
+
+### dbnsfp
+
+<i class="fas fa-exclamation"></i> required input: `--dbnsfp` [dbNSFP4.0a.zip](http://varianttools.sourceforge.net/Annotation/dbNSFP)  
+<i class="fas fa-exclamation-triangle"></i> required asset: *none*  
+<i class="fas fa-exclamation-circle"></i> required software: *none*
+
+The `dbnsfp` asset is the annotation database for non-synonymous SNPs.
+
+```
+wget ftp://dbnsfp:dbnsfp@dbnsfp.softgenetics.com/dbNSFP4.0a.zip
+refgenie build test/dbnsfp --dbnsfp dbNSFP4.0a.zip
+```
+
+## Derived assets you can build
+
+For many of the following derived assets, you will need the corresponding software to build the asset.  You can either [install software on a case-by-case basis natively](build.md#install-building-software-natively), or you can [build the assets using `docker`](build.md#building-assets-with-docker).
+
+### bowtie2 index
```

Comment:
let's be consistent with the headers format here: "bowtie2 index" vs "ensembl_rb"